AdvLoggerPkg/Library/AdvancedLoggerAccessLib/AdvancedLoggerAccessLib.c

```diff
@@ -40,10 +40,42 @@ CONST   CHAR8                 *AdvMsgEntryPrefix[ADVANCED_LOGGER_PHASE_CNT] = {
   "[TFA] ",
 };
 
-#define ADV_TIME_STAMP_FORMAT  "%2.2d:%2.2d:%2.2d.%3.3d : "
-#define ADV_TIME_STAMP_RESULT  "hh:mm:ss:ttt : "
-#define ADV_PHASE_ERR_FORMAT   "[%04X] "
-#define ADV_PHASE_MAX_SIZE     32
+// Define a structure to hold debug flag information
+typedef struct {
+  const char    *name;
+  UINT32        value;
+} DebugLevel;
+
+// Create an array of DebugLevel structures
+DebugLevel  debug_levels[] = {

```

Comment:
Please capitalize the variable and its type.

Reply:
- `DEBUG_LEVEL`
- `DebugLevels`

Reply:
Changed, thanks Michael.